bin/rose-make-docs

```diff
@@ -135,8 +135,8 @@ if [[ -z "${BUILDS}" ]]; then
 fi
 
 # Use virtualenv if present and requested or if we are in development mode.
-if [[ "${DEV_MODE}" != 'false' || "${VENV_MODE}" != false ]]; then
-    if [[ -z "${VENV_COMPLIANT}" ]]; then
+if [[ "${DEV_MODE}" != 'false' || ${VENV_MODE} != false ]]; then
+    if ${VENV_COMPLIANT}; then
```

Comment:
I believe you missed out the inversion here i.e. it should in fact be `if ! ...`. I checked by building without the `--venv` option on my home environment where a virtual environment is shown to be available and I get the 'if' statement block echo text.